### ISSUE TITLE
Lineage Input: Add support for multiple files (txtar)

### ISSUE THREAD

**Issue by joanlopez** [grafana/play-thema#9]
Would be really nice to have similar support like [Go Playground](https://go.dev/play/p/BWJ4dcUqVfT), where you can define your input by using multiple files, following the [txtar format](https://pkg.go.dev/golang.org/x/tools/txtar#hdr-Txtar_format).

Partially depends on #7
